Added  Google_Realistic.html

JMcrafter26 opened the pull request:
I added a new Google Sign in form.
This is the most realistic Google Sign in form out of these

JMcrafter26 committed:
Added  Google_Realistic.html

This is the most realistic Google Sign in form out of these

JMcrafter26 commented:
I seriously would add this, this is a big improvement

JMcrafter26 committed:
Added Google_Realistic_V2

This is the best google login remake by far, even animated with javascript and optimized for mobile and PC

JMcrafter26 commented:
This is the best google login remake by far, even animated with javascript and optimized for mobile and PC @RogueMaster

RogueMaster commented:
Cool! Good job! Thanks for the contribution! I think I merged this into my fork and firmware already 😀

JMcrafter26 commented:
> Cool! Good job! Thanks for the contribution! I think I merged this into my fork and firmware already 😀


@RogueMaster 
I made a second version, which is even better (With javascript and optimization).
This is why I pinged you, so you can add the second version as well.

Here is a Preview: [https://rawcdn.githack.com/JMcrafter26/flipper-zero-evil-portal/78cb63cddcc9955664e2e489a0598a534f4eb3e6/portals/Google_Realistic_V2.html](https://rawcdn.githack.com/JMcrafter26/flipper-zero-evil-portal/78cb63cddcc9955664e2e489a0598a534f4eb3e6/portals/Google_Realistic_V2.html)

JMcrafter26 committed:
Added Discord (By JMcrafter26)

RogueMaster commented:
I added the V2, however, Discord file exceeds the 20kb limit for the portals

JMcrafter26 commented:
> I added the V2, however, Discord file exceeds the 20kb limit for the portals
@RogueMaster 

Yes I know this, and I will work on it. The background svg is too big. (I accidentally added this unfinished file to the GitHub, I will work on it, when I have time. This will be a busy weekend)

JMcrafter26 committed:
Update Discord.html

JMcrafter26 committed:
Deleted Discord.html to avoid confusion

The discord html file is not ready yet. I will release a new version soon

SuperJakov commented:
Could you please make it so **inputs** and button **Next** arent bolded? Or just tell me where they are located so I can change font-weight

JMcrafter26 commented:
> Could you please make it so **inputs** and button **Next** arent bolded? Or just tell me where they are located so I can change font-weight

Just search for `.input` and `.next` In the array you will find the font weight

importerror commented:
Please merge this. Why is this still hanging?

JMcrafter26 commented:
> Please merge this. Why is this still hanging?

Idk what happened to bigbrodude (owner of this repo) he just doesn’t care for this repo. The last commit was over 6 months ago.
(I don’t have permission to merge this)

Dchostings commented:
I'm new to this how do add this to the flipper zero

JMcrafter26 commented:
> I'm new to this how do add this to the flipper zero

You have the firmware installed right?

Dchostings commented:
Yeah I've got extreme  firmware and in evil portal now it's just the demo extreme login

JMcrafter26 commented:
> Yeah I've got extreme firmware and in evil portal now it's just the demo extreme login

Ok, now do the following (this can be used to download other login screens too):
1. download the `.html` file ([example](https://github.com/JMcrafter26/flipper-zero-evil-portal/blob/main/portals/Google_Realistic.html))
2. Connect the flipper to your pc and start qFlipper
3. On the file browser tab, go to `app_assets/evil_portal/`, then if the folder `html` does not exist, create it and enter it
4. In the `html` folder, just drag and drop the downloaded file

Dchostings commented:
![IMG_20240514_194623866](https://github.com/bigbrodude6119/flipper-zero-evil-portal/assets/169309278/40a5a41f-c6df-4069-9a6b-16e05964889b)
Won't let me drag the file into here 🤔

JMcrafter26 commented:
> ![IMG_20240514_194623866](https://private-user-images.githubusercontent.com/169309278/330556765-40a5a41f-c6df-4069-9a6b-16e05964889b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3MTQ2OTgsIm5iZiI6MTcxNTcxNDM5OCwicGF0aCI6Ii8xNjkzMDkyNzgvMzMwNTU2NzY1LTQwYTVhNDFmLWM2ZGYtNDA2OS05YTZiLTE2ZTA1OTY0ODg5Yi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNFQxOTE5NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZGZiMGEzMWZlMjQzMzJiNWVkODY0NTU3ODNkMjEyOTc1ZGYwYjMzODQ4MjBkZWI5M2UzOWY1OGYzZjRjZTUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rkoDU3735Xxx9Dx1KJPEkCHg2xdQ7FREv3lb5_POaKw) Won't let me drag the file into here 🤔

Then just upload it

Dchostings commented:
Done it thanks for help I moved the files on me phone 👍

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: John M ***@***.***>
Sent: Tuesday, May 14, 2024 8:20:41 PM
To: bigbrodude6119/flipper-zero-evil-portal ***@***.***>
Cc: Dchostings ***@***.***>; Comment ***@***.***>
Subject: Re: [bigbrodude6119/flipper-zero-evil-portal] Added Google_Realistic.html (PR #60)


[IMG_20240514_194623866]<https://private-user-images.githubusercontent.com/169309278/330556765-40a5a41f-c6df-4069-9a6b-16e05964889b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3MTQ2OTgsIm5iZiI6MTcxNTcxNDM5OCwicGF0aCI6Ii8xNjkzMDkyNzgvMzMwNTU2NzY1LTQwYTVhNDFmLWM2ZGYtNDA2OS05YTZiLTE2ZTA1OTY0ODg5Yi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNFQxOTE5NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZGZiMGEzMWZlMjQzMzJiNWVkODY0NTU3ODNkMjEyOTc1ZGYwYjMzODQ4MjBkZWI5M2UzOWY1OGYzZjRjZTUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rkoDU3735Xxx9Dx1KJPEkCHg2xdQ7FREv3lb5_POaKw> Won't let me drag the file into here 🤔

Then just upload it

—
Reply to this email directly, view it on GitHub<https://github.com/bigbrodude6119/flipper-zero-evil-portal/pull/60#issuecomment-2110980783>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BILXIXQZ3QG4VCM3OWDDPGTZCJPYTAVCNFSM6AAAAAA3C6CFJSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMJQHE4DANZYGM>.
You are receiving this because you commented.Message ID: ***@***.***>

JMcrafter26 committed:
Add Google login page clone and guest Wi-Fi authentication page

- Created a revamped Google login page clone for educational purposes, including HTML structure, CSS styling, and JavaScript functionality for user login.
- Implemented a guest Wi-Fi authentication page with a simple design and functionality to authenticate devices for internet access.

JMcrafter26 committed:
Fixed redirect

JMcrafter26 committed:
Made a 50/50 chance of a redirect to the landing page

JMcrafter26 committed:
Enhance README with landing page image and disclaimer

Added an image of the landing page and updated the legal disclaimer.

JMcrafter26 committed:
Update README with link to revamped portals

Added note to redirect users to the revamped folder for updates.

JMcrafter26 committed:
Added Discord revamped plus

JMcrafter26 committed:
Merge branch 'main' of https://github.com/JMcrafter26/flipper-zero-evil-portal

JMcrafter26 commented:
@RogueMaster ive updated the google portal, it should now look identical to the real login page

JMcrafter26 commented:
[Preview of Revamped Plus](https://raw.githack.com/JMcrafter26/flipper-zero-evil-portal/main/portals_revamped/google/revamped_plus.html) | [Preview of Revamped](https://raw.githack.com/JMcrafter26/flipper-zero-evil-portal/main/portals_revamped/google/revamped.html)
